### PR TITLE
New structure 2growth

### PR DESCRIPTION
**Pull request by jvollhueter**
New structure allows the use of more than one growth model

### CODE REVIEW

File: ProjectLib/Project.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def argsToProject(self):
         self.iniNumpyRandomSeed()
         self.iniAbovegroundResourceConcept()
         self.iniBelowgroundResourceConcept()
-        self.iniPlantDynamicConcept()
+        # self.iniPlantDynamicConcept()
```
Comment (mcwimm) [inline ProjectLib/Project.py:22]: remove if not needed anymore

File: ProjectLib/XMLtoProject.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def __init__(self, **args):
         self.readProjectFile()
         self.addNumpyRandomSeed()
         self.addResourceConcept()
-        self.addPlantDynamicConcept()
+        # self.addPlantDynamicConcept()
```
Comment (mcwimm) [inline ProjectLib/XMLtoProject.py:27]: remove if not needed anymore

File: TimeLoopLib/DynamicTimeStep.py [hunk 3]
```diff
@@ -8,47 +8,47 @@
...
 
-class TreeDynamicTimeStepping:
+class DynamicTimeStep:
 
     def __init__(self, project):
         # Initialize concepts
         self.aboveground_resource_concept = project.getAbovegroundResourceConcept()
         self.belowground_resource_concept = project.getBelowgroundResourceConcept()
-        self.plant_dynamic_concept = project.getPlantDynamicConcept()
+        # self.plant_dynamic_concept = project.getPlantDynamicConcept()
```
Comment (mcwimm) [inline TimeLoopLib/DynamicTimeStep.py:17]: remove if not needed

File: TreeModelLib/BelowgroundCompetition/OGS/bin/AddElementQuality.exe
Comment (mcwimm) [inline TreeModelLib/BelowgroundCompetition/OGS/bin/AddElementQuality.exe:1]: all this OGS files need to be removed... maybe drop this commit?

**Review by mcwimm — CHANGES_REQUESTED** [review]
Cool! It seems to work. 
I rebased this branch onto the master and made a change to GroupPlanting.py. I think, this script can be shortened in general.
Also I added some working examples in Benchmark/Examples (didn't see your temp folder before).
However, it quite difficult to see the actual changes. That's why I would suggest to do the following steps before approval:

- [ ] finish and merge PR #236 
- [ ] update benchmark files to run all tests (at least PlantModules/Bettina and TreeOutput)

**Comment by jvollhueter** [comment]
At the moment the initialization of the mortality concept does not work anymore.

**Comment by jvollhueter** [comment]
@jvollhueter  argument time does not anymore get passed

**Review by mcwimm — APPROVED** [review]

**Comment by mcwimm** [comment]
the reference files for the benchmark are missing. sorry I recently added *.csv to gitignore to avoid uploading model outputs....

**Comment by jvollhueter** [comment]
Benchmark should now be completely included. xml tags are not allowed starting with a number, so had to change the structure a little bit. All .csv and .xml files are included now.

**Comment by jvollhueter** [comment]
> > Benchmark should now be completely included. xml tags are not allowed starting with a number, so had to change the structure a little bit. All .csv and .xml files are included now.
> 
> 👍 perfect

So we can merge it?

**Pull request merged**